includes/class-wc-payments-order-service.php

```diff
@@ -41,6 +41,13 @@ class WC_Payments_Order_Service {
 	 */
 	const CHARGE_ID_META_KEY = '_charge_id';
 
+	/**
+	 * Meta key used to store payment transaction Id.
+	 *
+	 * @const string
+	 */
+	const PAYMENT_TRANSACTION_ID_META_KEY = '_payment_transaction_id';
```

Comment:
most of the constant values are prefixed with `_wcpay`

Reply:
Yeah, all the new meta keys seem to follow that pattern. Fixed in https://github.com/Automattic/woocommerce-payments/pull/7945/commits/adc9a67919afb8df6bea59510f30566b49383c5b.